ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -359,7 +267,15 @@ function health_check() {
 
     #2. Check all cluster operators get stable and ready
     echo "Step #2: check all cluster operators get stable and ready"
-    wait_clusteroperators_continous_success
+    timeout 900s bash <<EOT
+until
+  oc wait clusteroperators --all --for='condition=Available=True' --timeout=30s && \
+  oc wait clusteroperators --all --for='condition=Progressing=False' --timeout=30s && \
+  oc wait clusteroperators --all --for='condition=Degraded=False' --timeout=30s;
```

Comment:
By default, it checks on all operators, right? If we don't want to check on a special operator, is it possible to skip it?

Reply:
should not be able to do